$NetBSD: patch-jira-wiki-markup.cabal,v 1.1 2020/01/16 15:02:50 pho Exp $

Don't build the executable "jira-wiki-markup". This package defines
both a library and an executable, and the executable depends on the
library. In this setup, either Cabal or GHC mistakenly embeds rpath to
the ${WRKDIR}, not the final installation path of the library.

The corresponding ticket has been closed without an actual fix, and
its authors have suggested using --enable-relocatable, even though the
option is supported only on Linux and MacOS X:
https://github.com/haskell/cabal/issues/1568

--- jira-wiki-markup.cabal.orig	2020-01-16 14:38:48.485995170 +0000
+++ jira-wiki-markup.cabal
@@ -52,6 +52,7 @@ library
   default-extensions:  OverloadedStrings
 
 executable jira-wiki-markup
+  buildable:           False
   hs-source-dirs:      app
   main-is:             Main.hs
 
